### ISSUE TITLE
Smart Duplication Detection

### ISSUE THREAD

**Issue by sr229** [ClarityCafe/Aya#8]
We can use a AI Service for this. 

We train a model everytime a new image is uploaded, but its only scheduled per 2 hours to train from the entire collection or just what got uploaded new.

- Post exists:
    - block/warn user.

**Comment by DonaldTsang** [comment]
Also what kind of hashing will you use? There are many like pHash, aHash, dHash and wHash (with varying resolutions)

**Comment by sr229** [comment]
I don't think Hashing can solve duplication since anyone can just waifu2x or change the format and it'll result in a different hashing.

**Comment by DonaldTsang** [comment]
No, pHash and the like hashes the LOOKS of an image, not the data.
https://en.wikipedia.org/wiki/Perceptual_hashing

**Comment by sr229** [comment]
I'll see what I can do to adapt the hashing standard, but this is on hold for now since its in the FVP log.
